### PR TITLE
store: add refresh_store.sh

### PR DESCRIPTION
**Pull request by OniriCorpe**

### CODE REVIEW

File: store/refresh_store.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+workdir=$(realpath "$(dirname "$0")")
+log=$workdir/refresh_store.log
+
+cd "$workdir"
+date >> "$log"
+git pull &>/dev/null
+cat cron | sed "s@__BASEDIR__@$workdir@g" > /etc/cron.d/refresh_store
```
Comment (alexAubin) [inline store/refresh_store.sh:9]: Uuuuuh :thinking: 

Not sure to understand, is this script supposed to be ran from the cron job :thinking:  ? (https://github.com/YunoHost/apps/blob/master/cron)

**Comment by Psycojoker** [comment]
Maybe also updating the documentation would be a good idea?

**Comment by OniriCorpe** [comment]
use `inotifywait`?

**Comment by tituspijean** [comment]
or systemd `.path` ?

**Comment by alexAubin** [comment]
(Will redo something in the coming days / weeks)

**Pull request closed without merging**